app/pages/nonsipp/landorproperty/IsLesseeConnectedPartyPage.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.landorproperty
+
+import config.Refined.Max5000
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+
+case class IsLesseeConnectedPartyPage(srn: Srn, index: Max5000) extends QuestionPage[Boolean] {
+
+  override def path: JsPath = JsPath \ toString
+
+  override def toString: String = "isLesseeConnectedParty"
```

Comment:
Need to be a path in land or property structure. Could you please also update the mapping table?

Reply:
Sorry forgot to add that,  I have just updated both

Reply:
Thanks for changing it @narendrapaduchuri.... would it be ok to use the constants defined instead of hardcoded values please, as for example in WhyDoesSchemeHoldLandPropertyPage?  This is more for a future reference than any change for this functionality.